eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/steps/processors/VariantToVariantContextProcessor.java

```diff
@@ -61,8 +67,11 @@ public VariantContext process(IVariant variant) {
         }
         String[] allelesArray = getAllelesArray(variant);
 
+        String contig = variant.getChromosome();
+        String sequenceName = getSequenceName(contig);
```

Comment:
Maybe these could be merged in one line? One less local variable to optimize, and `contig` is not used anyway.